src/index.js

```diff
@@ -31,6 +32,10 @@ export default class Transition extends Component {
     this.start();
   }
 
+  componentWillUnmount = () => {
+    this.mounted = false;
```

Comment:
Would calling `this.timer.stop()` here achieve the same as tracking the mounted state?

Reply:
I couldn't tell from the code, would that stop any additional `setState` functions from firing? If so that would probably be a little more elegant than manually tracking the mount state.

Reply:
I believe so since the mounted flag is set in the same tick as the timer tick. I'll test it out tomorrow.